wordpress-mobile/WordPress-iOS
# Issue/10971 implement missing media actions

Fixes #10971 

This PR implements some stubbed media upload checks in order to properly validate the post media states in the GB editor.

To test:
 - Start a new post
 - Publish it some text
 - Edit it again
 - Add media to it
 - While the media is uploading
 - Try to tap update on the top right.
 - Check that an alert shows up saying that media is uploading and you need to wait.
